DepartmentDetails/src/com/fsd/group4/departmentinterfaces/DepartmentUtilities.java

```diff
@@ -0,0 +1,12 @@
+package com.fsd.group4.departmentinterfaces;
+
+public interface DepartmentUtilities {
```

Comment:
Can we be consistent on naming convention as followed for other interfaces e.g. IDepartmentUtilities